apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/components/amount_selection/BaseAmountBox.java

```diff
@@ -130,7 +130,7 @@ void reset() {
     }
 
     private static class View extends bisq.desktop.common.view.View<HBox, Model, Controller> {
-        private Subscription amountListener, isBtcListener, codeListener;
+        private Subscription amountListenerPin, isBtcListenerPin, codeListenerPin;
```

Comment:
Should be `amountPin` - you can see that pattern used everywhere in our UI code.

Reply:
I will fix it myself in a upcoming PR